src/components/modal/base/modal-inner-content.scss

```diff
@@ -6,7 +6,7 @@ necessary to prevent modal-inner-content's margins from
 unexpectedly collapsing with a parent
 */
 .modal-inner-clear {
-    overflow: auto;
+    float: left;
```

Comment:
better method of clearing margins, which allows validation errors to overflow box